jbpm-bpmn2/src/test/resources/BPMN2-CallActivityWithBoundaryEvent.bpmn2

```diff
@@ -39,7 +39,7 @@
     <boundaryEvent id="BoundaryEvent_1" name="Boundary event" attachedToRef="_2">
       <outgoing>SequenceFlow_2</outgoing>
       <timerEventDefinition id="TimerEventDefinition_1">
-        <timeDuration xsi:type="tFormalExpression" id="_FormalExpression_2">2s</timeDuration>
+        <timeDuration xsi:type="tFormalExpression" id="_FormalExpression_2"><![CDATA[#{java.lang.System.getProperty("delayInProductionEnv","PT5s")}]]></timeDuration>
```

Comment:
is there any reason to extend it from 2s to 5s?

Reply:
+1, can we leave it as it was?

Reply:
done